books/projects/x86isa/machine/instructions/pcmp.lisp

```diff
@@ -0,0 +1,149 @@
+; X86ISA Library
```

Comment:
This new instruction definition implements `pcmpeq{b,w,q}`, but it is only used for `pcmpeq{w,q}`, I think because I didn't notice `pcmpeqb` was already implemented until I was done. I think it would be reasonable to use this implementation for `pcmpeqb` as well and then remove the other definition.

Reply:
Good idea. The new definition is simpler, and more uniform.